### PR TITLE
Contributing guidelines

### PR DESCRIPTION
**Pull request by masinter**
Please review and make suggestions. I started with one of the samples (Ruby on Rails) as recommended in GitHub.
Do we need different guidelines for Maiko and Medley?  Should we put the approval policy here or elsewhere (I don't think we have consensus yet on that).
This is one of the steps for https://github.com/Interlisp/medley/issues/111

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# How to contribute to Medley Interlisp
```
Comment (waywardmonkeys) [inline CONTRIBUTING.md:1]: Since this is in the Maiko repository, I feel like this should have some indication here.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+## Reporting a bug or feature request
+* Ensure the bug was not already reported by searching on GitHub under [Issues](https://github.com/Interlisp/medley/issues) or [Discussions](https://github.com/Interlisp/medley/discussions).
+Discussions are for questions or topics where there is some disagreement or uncertainty about the "right" direction.
+* If you're unable to find a discussion or open issue addressing the problem, open a new one. Be sure to include a title 
+and clear description, as much relevant information as possible, 
+and a code sample or an executable test case demonstrating the expected behavior that is not occurring.
+
+## Did you write a patch that fixes a bug?
+* Open a new GitHub pull request with the patch
```
Comment (waywardmonkeys) [inline CONTRIBUTING.md:16]: Missing a period at the end of this. Should probably also link 'GitHub pull request' to https://github.com/Interlisp/maiko/pulls

**Review by waywardmonkeys — CHANGES_REQUESTED** [review]

**Comment by nbriggs** [comment]
I think we may need to have separate versions of the file for the Lisp side and the emulator code.
If you're reporting a problem in Lisp (or you don't know whether it's a bug in the Lisp code or the emulator) you should include which sysout you're running in, and the emulator version (we should make this retrievable from the Common Lisp environment info), and the OS and processor architecture (so we know big-/little- endian -- Solaris 11 could be either; 32- or 64-bit).  On the Lisp side, the LOADEDFILELST and FILELST have useful info.

**Comment by masinter** [comment]
@nbriggs I will propose some[issue templates](https://docs.github.com/en/free-pro-team@latest/github/building-a-strong-community/configuring-issue-templates-for-your-repository) for reporting errors that will ask for appropriate information

**Comment by nbriggs** [comment]
OK.   Have a look at https://www.freebsd.org/doc/en_US.ISO8859-1/articles/problem-reports/article.html as well, it's not *directly* applicable to our code but it makes a lot of good points.

**Comment by nbriggs** [comment]
Also -- https://github.com/ncovercash/reporting-bugs-effectively/blob/master/ENGLISH.md

**Comment by masinter** [comment]
I just read about setting issue templates and other community guidelines for the whole Interlisp organization by setting up a .github repo, and I'm going to do that.

**Comment by nbriggs** [comment]
OK, could you spell check the document before committing it?

**Pull request closed without merging**